packages/at_onboarding_cli/lib/src/activate_cli/activate_cli.dart

```diff
@@ -26,24 +26,22 @@ Future<void> main(List<String> arguments) async {
 
   if (argResults.wasParsed('help')) {
     print(parser.usage);
-    exit(0);
+    return;
```

Comment:
How do we ensure that the return statements here and on lines 35 and 41 result in a non-zero exit status?

Reply:
As discussed, will refactor the code to throw an exception from here. And return a non-zero exit code from the caller of this method.

Reply:
This change is done